### ISSUE TITLE
Packet 0x54 Announcement

### ISSUE THREAD

**Issue by Killer233** [LiamKarlMitchell/InfiniteSky#152]
Map paldo island open in 4 minutes

````
54 2D 00 00 00 04 00 00 00 FF FF FF FF 88 B8 DF
77 92 BA DF 77 00 00 00 00 00 10 40 00 5C 02 08
00 13 01 00 00 01 00 00 00 00 00 00 00 04 8C E0
00 AB BA DF 77 00 00 00 00 AC BD DF 77 D6 16 E0
77 00 00 00 00 CB 74 44 07 1B 00 00 00 C1 00 00
00 01 00 00 00 01 00 00 00 00 00 00 00 E4 FD 12
00 3E 3D DF 77 B0 FF 12 00                     
````

**Comment by LiamKarlMitchell** [comment]
[Wargo Valley] is open
````
54 11 00 00 00 02 00 00 00 FF FF FF FF 88 B8 DF 
77 92 BA DF 77 00 00 00 00 00 10 40 00 90 01 0F 
00 13 01 00 00 01 00 00 00 00 00 00 00 FC 74 DF 
00 AB BA DF 77 00 00 00 00 AC BD DF 77 D6 16 E0 
77 00 00 00 00 23 A3 0F 30 68 00 00 00 B5 00 00 
00 01 00 00 00 01 00 00 00 00 00 00 00 E4 FD 12 
00 3E 3D DF 77 B0 FF 12 00
````

**Comment by Retain94** [comment]
Some codes will not work if you have the minutes like for example:
[-1] minutes, [Guangwon Formation] will be opened has a [-1] but if u switch the code
54 XX XX XX XX 02 to 54 XX XX XX 04
the code will not do any action, so I'm guessing there its 2 different ways to use the [Minutes] with the formations

**Comment by LiamKarlMitchell** [comment]
That sounds like a possibility there is a lot of data in this packet that
is one way or another depending on the type.

**Comment by LiamKarlMitchell** [comment]
[Start the Nangi Track] The [Nangi Track] will begin in [5] minutes.
[Start the Nangi Track] The [Nangi Track] will begin in [4] minutes.
[Start the Nangi Track] The [Nangi Track] will begin in [4] minutes.
[Start the Nangi Track] The [Nangi Track] will begin in [4] minutes.

**Comment by credmann15** [comment]
i agree however remove number 8. People like to chill in the track no idea why but they do lol.

**Comment by LiamKarlMitchell** [comment]
# 8 could be used if we wanted to instance the map by taking it online/offline but ehh sure i guess they can stay.

**Comment by LiamKarlMitchell** [comment]
Just noticed lots of messages missed here because the ID was not incremented in hex properly...

Message Type
0B Faction Stone Destroy

First Integer is stone that was destroyed.
0 Guanyin Stone Destroy
1 Fujin Stone Destroy
2 Jinong Stone Destroy
3 Yaoguai Stone Destroy

Second Integer is the clan that destroyed the stone.
0 Guanyin
1 Fujin
2 Jinong

**Comment by LiamKarlMitchell** [comment]
00 [Clan] [Name]has used initiated [Formation]!
01 The initiation of [Formation] has been stopped.
02 [Formation] will be opened [Minutes]n minutes.
03 
04 [Clan] [Name]has successfully opened [Formation]!
05 [Clan] [Name]has initiated the [Formation] Battle!
06 The [Formation] Battle has been stopped.
07 [Formation] Battle [Minutes]in minutes.
08 [Clan] [Name]has won the [Formation] Battle!
09 [Guild]has [Name]created a guild.
0A [Name]You have joined the guild.
0B [] [Faction Stone] Destroy!
0C Garbage Text
0D [] [Faction Stone] Attack!
0E [Faction Stone] Recover!
0F [Minutes] minutes. The [Samji Island] entrance will be open.
10 [Samji Island] is open
11 [Samji Island] is closed
12 
13 [] [Name] has initiated [Formation]!
14 The initiation of [Formation] has been stopped.
15 [Minutes] minutes. [Formation] will be open.
16 [] [Name]has successfully opened [Formation]!
17 [Minutes] minutes. [Formation] will be closed.
18 [Formation] is closed
19 []The guild has been closed down.
1A [Minutes] minutes. The [Formation] Battle will start.
1B You can start the [Formation] Battle.
1C []has been expelled.
1D 
1E []has been fired.
1F []has revived the title.
20 
21 
22 [Minutes] minutes. [Sangun Island] will be opened.
23 [Sangun Island] has opened!
24 [Clan] has entered [Formation]!
25 [] [Name] has initiated the [Formation]!
26 The initiation of [Formation] has been stopped.
27 [Minutes] minutes. [Formation] will be opened.
28 [] [Name] has successfully opened the [Formation]!
29 
30
31 
32
33 [Minutes] minutes. [Paldo Island] will be opened.
34 [Paldo Island] is open!
35 [Clan] has entered [Formation]!
36 [Clan] [Name]has initiated [Formation]!
37 [] [Name] has successfully opened the [Formation]!
38 The initiation of [Formation] has been stopped.
39 [Minutes] minutes. [Formation] will be opened.
3A [Clan] [Name] has successfully opened [Formation]!
3B [Formation] is closed down!
3C No clan has occupied [Formation].
3D 
3E
3F <Crashes game>
